flaggers/unobservedStop.py

```diff
@@ -16,15 +16,15 @@ def flag(self, data):
 
 		Args:
 			data (Object): data row from full dataset fetched from the db
+			config (Object): contains config vars
 
 		Returns: 
 			list: either empty or containing UNOBSERVED_STOP Flag
 
 		"""
 
 		#maxDistance specifies max distance away from the stop at which we start marking stop as an unobserved
-		#Instead of being hardcoded, in the future substitute to .env variable
-		maxDistance = 50
+		maxDistance = config.get_value("unobserved_stop_distance")
 
```

Comment:
If this doesn't store the config variable, I think it should default to some value just so everything that assumes it is a value does not fail as well.

Reply:
Good point, I'll make a change